### ISSUE TITLE
Telegram freezes if it need to send or receive a call

### ISSUE THREAD

**Issue by banderlog** [telegramdesktop/tdesktop#10421]
### Steps to reproduce
1. Try to call to someone or receive a call

### Expected behaviour

You shoud be able to do so

### Actual behaviour

Telegram freezes with next console output:
```
[ALSOFT] (EE) Failed to set real-time priority for thread: Operation not permitted (1)
error: : cannot open
error: : cannot open
error: : cannot open
Invalid return value 0 for stream protocol
Invalid return value 0 for stream protocol
[ALSOFT] (EE) Failed to set real-time priority for thread: Operation not permitted (1)
QTextCursor::setPosition: Position '8388607' out of range
QTextCursor::setPosition: Position '8388607' out of range
[ALSOFT] (EE) Failed to set real-time priority for thread: Operation not permitted (1)
[ALSOFT] (EE) Failed to set real-time priority for thread: Operation not permitted (1)

```

### Configuration

Ubuntu 18.04.5 LTS

**Version of Telegram Desktop:**
2.5.8

**Installation source (Linux Only)** - the official website / GitHub releases / flatpak / snap / distribution package:
the official website

**Used theme**:
Tinted

**Comment by granger75** [comment]
Still present in 2.5.9 .
Mageia7.1 - kernel 5.10.14 - Qt 5.12.6 - Plasma 5.15.4 on Samsung RC730
BTW: 2.5.8 was working as expected....

**Comment by BaderSZ** [comment]
Can confirm. Telegram desktop freezes when I try to join voice chats. 

```
Telegram Desktop 2.5.9
OS: Fedora 32 ThirtyTwo
Kernel: x86_64 Linux 5.10.16-100.fc32.x86_64
CPU: Intel Core i5-3320M @ 4x 3.3GHz
```

**Comment by BaderSZ** [comment]
Telegram -debug output:

```
(field_trial.cc:140): Setting field trial string:WebRTC-Audio-Allocation/min:32kbps,max:32kbps/WebRTC-Audio-OpusMinPacketLossRate/Enabled-1/WebRTC-PcFactoryDefaultBitrates/min:32kbps,start:32kbps,max:32kbps/
(audio_device_buffer.cc:65): AudioDeviceBuffer::ctor
(audio_device_buffer.cc:181): SetRecordingSampleRate(48000)
(audio_device_buffer.cc:187): SetPlayoutSampleRate(48000)
(audio_device_buffer.cc:201): SetRecordingChannels(1)
(audio_device_buffer.cc:207): SetPlayoutChannels(2)
(audio_processing_impl.cc:274): Injected APM submodules:
Echo control factory: 0
Echo detector: 0
Capture analyzer: 1
Capture post processor: 0
Render pre processor: 0
(audio_processing_impl.cc:531): AudioProcessing::ApplyConfig: AudioProcessing::Config{ pipeline: {maximum_internal_processing_rate: 48000, multi_channel_render: 0, multi_channel_capture: 0}, pre_amplifier: { enabled: 0, fixed_gain_factor: 1 }, high_pass_filter: { enabled: 1 }, echo_canceller: { enabled: 0, mobile_mode: 0, enforce_high_pass_filtering: 1 }, noise_suppression: { enabled: 1, level: High }, transient_suppression: { enabled: 0 }, voice_detection: { enabled: 0 }, gain_controller1: { enabled: 0, mode: AdaptiveAnalog, target_level_dbfs: 3, compression_gain_db: 9, enable_limiter: 1, analog_level_minimum: 0, analog_level_maximum: 255 }, gain_controller2: { enabled: 0, fixed_digital: { gain_db: 0}, adaptive_digital: { enabled: 0, level_estimator: { vad_probability_attack: 1, type: Rms, adjacent_speech_frames_threshold: 1, initial_saturation_margin_db: 20, extra_saturation_margin_db: 2}, gain_applier: { adjacent_speech_frames_threshold: 1, max_gain_change_db_per_second: 3, max_output_noise_level_dbfs: -50 } }, residual_echo_detector: { enabled: 1 }, level_estimation: { enabled: 0 }}}
(webrtc_voice_engine.cc:265): WebRtcVoiceEngine::WebRtcVoiceEngine
(webrtc_video_engine.cc:628): WebRtcVideoEngine::WebRtcVideoEngine()
(webrtc_voice_engine.cc:287): WebRtcVoiceEngine::Init
(adm_helpers.cc:77): Failed to set stereo recording mode.
(audio_device_buffer.cc:82): RegisterAudioCallback
(webrtc_voice_engine.cc:383): WebRtcVoiceEngine::ApplyOptions: AudioOptions {aec: 1, agc: 1, ns: 1, hf: 1, swap: 0, audio_jitter_buffer_max_packets: 200, audio_jitter_buffer_fast_accelerate: 0, audio_jitter_buffer_min_delay_ms: 0, audio_jitter_buffer_enable_rtx_handling: 0, typing: 1, experimental_agc: 0, experimental_ns: 0, residual_echo_detector: 1, }
(webrtc_voice_engine.cc:492): Stereo swapping enabled? 0
(webrtc_voice_engine.cc:497): NetEq capacity is 200
(webrtc_voice_engine.cc:503): NetEq fast mode? 0
(webrtc_voice_engine.cc:509): NetEq minimum delay is 0
(webrtc_voice_engine.cc:515): NetEq handle reordered packets? 0
(webrtc_voice_engine.cc:535): Experimental ns is enabled? 0
(webrtc_voice_engine.cc:586): NS set to 1
(webrtc_voice_engine.cc:590): Typing detection is enabled? 1
(audio_processing_impl.cc:531): AudioProcessing::ApplyConfig: AudioProcessing::Config{ pipeline: {maximum_internal_processing_rate: 48000, multi_channel_render: 0, multi_channel_capture: 0}, pre_amplifier: { enabled: 0, fixed_gain_factor: 1 }, high_pass_filter: { enabled: 1 }, echo_canceller: { enabled: 1, mobile_mode: 0, enforce_high_pass_filtering: 1 }, noise_suppression: { enabled: 1, level: High }, transient_suppression: { enabled: 0 }, voice_detection: { enabled: 1 }, gain_controller1: { enabled: 1, mode: AdaptiveAnalog, target_level_dbfs: 3, compression_gain_db: 9, enable_limiter: 1, analog_level_minimum: 0, analog_level_maximum: 255 }, gain_controller2: { enabled: 0, fixed_digital: { gain_db: 0}, adaptive_digital: { enabled: 0, level_estimator: { vad_probability_attack: 1, type: Rms, adjacent_speech_frames_threshold: 1, initial_saturation_margin_db: 20, extra_saturation_margin_db: 2}, gain_applier: { adjacent_speech_frames_threshold: 1, max_gain_change_db_per_second: 3, max_output_noise_level_dbfs: -50 } }, residual_echo_detector: { enabled: 1 }, level_estimation: { enabled: 0 }}}
(agc_manager_direct.cc:69): [agc] GetMinMicLevel
(agc_manager_direct.cc:73): [agc] Using default min mic level: 12
(peer_connection_factory.cc:354): Using default network controller factory
(bitrate_prober.cc:72): Bandwidth probing enabled, set to inactive
(cpu_info.cc:53): Available number of cores: 4
(aimd_rate_control.cc:113): Using aimd rate control with back off factor 0.85
(remote_bitrate_estimator_single_stream.cc:72): RemoteBitrateEstimatorSingleStream: Instantiating.
(remote_estimator_proxy.cc:50): Maximum interval between transport feedback RTCP messages (ms): 250
(openssl_key_pair.cc:38): Making key pair
(rtp_transmission_manager.cc:186): Adding audio transceiver in response to a call to AddTrack.
(AudioDeviceHelper.cpp:35): setAudioInputDevice(): SetRecordingDevice(kDefaultCommunicationDevice) success.
(AudioDeviceHelper.cpp:81): setAudioOutputDevice(): SetPlayoutDevice(kDefaultCommunicationDevice) success.
(openssl_key_pair.cc:91): Returning key pair
(openssl_certificate.cc:59): Making certificate for WebRTC
[ALSOFT] (EE) Failed to set real-time priority for thread: Operation not permitted (1)
(openssl_certificate.cc:109): Returning certificate
(audio_device_buffer.cc:187): SetPlayoutSampleRate(48000)
(audio_device_buffer.cc:207): SetPlayoutChannels(2)
(audio_device_buffer.cc:99): StartPlayout
(thread.cc:688): Message took 57ms to dispatch. Posted from: GroupInstanceImpl@/usr/src/tdesktop/Telegram/ThirdParty/tgcalls/tgcalls/group/GroupInstanceImpl.cpp:2029
(AudioDeviceHelper.cpp:35): setAudioInputDevice(default): SetRecordingDevice(kDefaultCommunicationDevice) success.
(audio_device_buffer.cc:140): StopPlayout
(audio_device_buffer.cc:146): total playout time: 1
[ALSOFT] (EE) Context 0x7f906007bb10 current for thread being destroyed!
(AudioDeviceHelper.cpp:81): setAudioOutputDevice(default): SetPlayoutDevice(kDefaultCommunicationDevice) success.
[ALSOFT] (EE) Failed to set real-time priority for thread: Operation not permitted (1)
(audio_device_buffer.cc:187): SetPlayoutSampleRate(48000)
(audio_device_buffer.cc:207): SetPlayoutChannels(2)
(audio_device_buffer.cc:99): StartPlayout
(used_ids.h:55): Duplicate id found. Reassigning from 110 to 127
(used_ids.h:55): Duplicate id found. Reassigning from 106 to 126
(used_ids.h:55): Duplicate id found. Reassigning from 111 to 125
(GroupInstanceImpl.cpp:1466): ----- setLocalDescription join -----
(GroupInstanceImpl.cpp:1467): v=0
o=- 7485152035293971574 2 IN IP4 127.0.0.1
s=-
t=0 0
a=group:BUNDLE 0
a=msid-semantic: WMS
m=audio 9 UDP/TLS/RTP/SAVPF 111 110
c=IN IP4 0.0.0.0
a=rtcp:9 IN IP4 0.0.0.0
a=ice-ufrag:l8iN
a=ice-pwd:REDACTED
a=ice-options:trickle
a=fingerprint:sha-256 REDACTED
a=setup:actpass
a=mid:0
a=extmap:1 urn:ietf:params:rtp-hdrext:ssrc-audio-level
a=extmap:2 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time
a=extmap:3 http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01
a=extmap:4 urn:ietf:params:rtp-hdrext:sdes:mid
a=extmap:5 urn:ietf:params:rtp-hdrext:sdes:rtp-stream-id
a=extmap:6 urn:ietf:params:rtp-hdrext:sdes:repaired-rtp-stream-id
a=recvonly
a=rtcp-mux
a=rtpmap:111 opus/48000/2
a=rtcp-fb:111 transport-cc
a=fmtp:111 minptime=10;useinbandfec=1
a=rtpmap:110 telephone-event/48000

(GroupInstanceImpl.cpp:1468): -----
(p2p_transport_channel.cc:532): Set backup connection ping interval to 25000 milliseconds.
(p2p_transport_channel.cc:541): Set ICE receiving timeout to 2500 milliseconds
(p2p_transport_channel.cc:548): Set ping most likely connection to 1
(p2p_transport_channel.cc:555): Set stable_writable_connection_ping_interval to 2500
(p2p_transport_channel.cc:586): Set regather_on_failed_networks_interval to 300000
(p2p_transport_channel.cc:593): Set receiving_switching_delay to 1000
(jsep_transport_controller.cc:1008): Creating DtlsSrtpTransport.
(dtls_srtp_transport.cc:62): Setting RTCP Transport on 0 transport 0
(dtls_srtp_transport.cc:67): Setting RTP Transport on 0 transport 6c003680
(p2p_transport_channel.cc:466): Set ICE ufrag: l8iN pwd: REDACTED on transport 0
(webrtc_voice_engine.cc:1611): Setting voice channel options: AudioOptions {audio_jitter_buffer_max_packets: 200, audio_jitter_buffer_fast_accelerate: 1, audio_jitter_buffer_min_delay_ms: 0, audio_jitter_buffer_enable_rtx_handling: 0, }
(webrtc_voice_engine.cc:383): WebRtcVoiceEngine::ApplyOptions: AudioOptions {audio_jitter_buffer_max_packets: 200, audio_jitter_buffer_fast_accelerate: 1, audio_jitter_buffer_min_delay_ms: 0, audio_jitter_buffer_enable_rtx_handling: 0, }
(webrtc_voice_engine.cc:497): NetEq capacity is 200
(webrtc_voice_engine.cc:503): NetEq fast mode? 1
(webrtc_voice_engine.cc:509): NetEq minimum delay is 0
(webrtc_voice_engine.cc:515): NetEq handle reordered packets? 0
(webrtc_voice_engine.cc:535): Experimental ns is enabled? 0
(audio_processing_impl.cc:531): AudioProcessing::ApplyConfig: AudioProcessing::Config{ pipeline: {maximum_internal_processing_rate: 48000, multi_channel_render: 0, multi_channel_capture: 0}, pre_amplifier: { enabled: 0, fixed_gain_factor: 1 }, high_pass_filter: { enabled: 1 }, echo_canceller: { enabled: 1, mobile_mode: 0, enforce_high_pass_filtering: 1 }, noise_suppression: { enabled: 1, level: High }, transient_suppression: { enabled: 0 }, voice_detection: { enabled: 1 }, gain_controller1: { enabled: 1, mode: AdaptiveAnalog, target_level_dbfs: 3, compression_gain_db: 9, enable_limiter: 1, analog_level_minimum: 0, analog_level_maximum: 255 }, gain_controller2: { enabled: 0, fixed_digital: { gain_db: 0}, adaptive_digital: { enabled: 0, level_estimator: { vad_probability_attack: 1, type: Rms, adjacent_speech_frames_threshold: 1, initial_saturation_margin_db: 20, extra_saturation_margin_db: 2}, gain_applier: { adjacent_speech_frames_threshold: 1, max_gain_change_db_per_second: 3, max_output_noise_level_dbfs: -50 } }, residual_echo_detector: { enabled: 1 }, level_estimation: { enabled: 0 }}}
(webrtc_voice_engine.cc:1629): Set voice channel options. Current options: AudioOptions {audio_jitter_buffer_max_packets: 200, audio_jitter_buffer_fast_accelerate: 1, audio_jitter_buffer_min_delay_ms: 0, audio_jitter_buffer_enable_rtx_handling: 0, }
(channel.cc:145): Created channel: {mid: 0, media_type: audio}
(rtp_demuxer.cc:144): Added sink = 6014dc08 for criteria {mid: 0, rsid: <empty>, ssrcs: [], payload_types = []}
(sdp_offer_answer.cc:2458): Session: 7485152035293971574 Old state: stable New state: have-local-offer
(channel.cc:905): Setting local voice description for {mid: 0, media_type: audio}
(webrtc_voice_engine.cc:1471): WebRtcVoiceMediaChannel::SetRecvParameters: {codecs: [AudioCodec[111:opus:48000:0:2], AudioCodec[110:telephone-event:48000:0:1]], extensions: [{uri: urn:ietf:params:rtp-hdrext:ssrc-audio-level, id: 1}, {uri: http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time, id: 2}, {uri: http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01, id: 3}, {uri: urn:ietf:params:rtp-hdrext:sdes:mid, id: 4}, {uri: urn:ietf:params:rtp-hdrext:sdes:rtp-stream-id, id: 5}, {uri: urn:ietf:params:rtp-hdrext:sdes:repaired-rtp-stream-id, id: 6}]}
(webrtc_voice_engine.cc:1639): Setting receive voice codecs.
(channel.cc:896): Changing voice state, recv=0 send=0 for {mid: 0, media_type: audio}
(rtp_demuxer.cc:240): Removed sink = 6014dc08 bindings
(rtp_demuxer.cc:144): Added sink = 6014dc08 for criteria {mid: 0, rsid: <empty>, ssrcs: [], payload_types = [110, 111, ]}
(basic_port_allocator.cc:375): Start getting ports with turn_port_prune_policy 0
(basic_port_allocator.cc:111): Filtered out ignored networks:
(basic_port_allocator.cc:113): Net[lo:0:0:0:x:x:x:x:x/128:Loopback:id=2]
(basic_port_allocator.cc:113): Net[lo:127.0.0.x/8:Loopback:id=1]
(basic_port_allocator.cc:861): Network manager has started
(basic_port_allocator.cc:111): Filtered out ignored networks:
(basic_port_allocator.cc:113): Net[lo:0:0:0:x:x:x:x:x/128:Loopback:id=2]
(basic_port_allocator.cc:113): Net[lo:127.0.0.x/8:Loopback:id=1]
(basic_port_allocator.cc:776): Allocate ports on 3 networks
(basic_port_allocator.cc:1360): Net[wlp3s0:2a02:908:1013:x:x:x:x:x/64:Unknown:id=4]: Allocation Phase=Udp
(port.cc:186): Port[6c039a50::1:0:local:Net[wlp3s0:2a02:908:1013:x:x:x:x:x/64:Unknown:id=4]]: Port created with network cost 50
(basic_port_allocator.cc:885): Adding allocated port for 0
(basic_port_allocator.cc:906): Port[6c039a50:0:1:0:local:Net[wlp3s0:2a02:908:1013:x:x:x:x:x/64:Unknown:id=4]]: Added port to allocator
(basic_port_allocator.cc:924): Port[6c039a50:0:1:0:local:Net[wlp3s0:2a02:908:1013:x:x:x:x:x/64:Unknown:id=4]]: Gathered candidate: Cand[:1485711723:1:udp:2122262783:[2a02:908:1013:x:x:x:x:x]:33808:local::0:l8iN:j+YbUepJBqspezc2DEz0GlZe:4:50:0]
(basic_port_allocator.cc:957): Port[6c039a50:0:1:0:local:Net[wlp3s0:2a02:908:1013:x:x:x:x:x/64:Unknown:id=4]]: Port ready.
(basic_port_allocator.cc:1068): Port[6c039a50:0:1:0:local:Net[wlp3s0:2a02:908:1013:x:x:x:x:x/64:Unknown:id=4]]: Port completed gathering candidates.
(basic_port_allocator.cc:1360): Net[wlp3s0:2a02:908:1013:x:x:x:x:x/128:Unknown:id=5]: Allocation Phase=Udp
(port.cc:186): Port[6c03ad10::1:0:local:Net[wlp3s0:2a02:908:1013:x:x:x:x:x/128:Unknown:id=5]]: Port created with network cost 50
(basic_port_allocator.cc:885): Adding allocated port for 0
(basic_port_allocator.cc:906): Port[6c03ad10:0:1:0:local:Net[wlp3s0:2a02:908:1013:x:x:x:x:x/128:Unknown:id=5]]: Added port to allocator
(basic_port_allocator.cc:924): Port[6c03ad10:0:1:0:local:Net[wlp3s0:2a02:908:1013:x:x:x:x:x/128:Unknown:id=5]]: Gathered candidate: Cand[:2082451711:1:udp:2122197247:[2a02:908:1013:x:x:x:x:x]:53424:local::0:l8iN:j+YbUepJBqspezc2DEz0GlZe:5:50:0]
(basic_port_allocator.cc:957): Port[6c03ad10:0:1:0:local:Net[wlp3s0:2a02:908:1013:x:x:x:x:x/128:Unknown:id=5]]: Port ready.
(basic_port_allocator.cc:1068): Port[6c03ad10:0:1:0:local:Net[wlp3s0:2a02:908:1013:x:x:x:x:x/128:Unknown:id=5]]: Port completed gathering candidates.
(basic_port_allocator.cc:1360): Net[wlp3s0:192.168.0.x/24:Unknown:id=3]: Allocation Phase=Udp
(port.cc:186): Port[6c03be30::1:0:local:Net[wlp3s0:192.168.0.x/24:Unknown:id=3]]: Port created with network cost 50
(basic_port_allocator.cc:885): Adding allocated port for 0
(basic_port_allocator.cc:906): Port[6c03be30:0:1:0:local:Net[wlp3s0:192.168.0.x/24:Unknown:id=3]]: Added port to allocator
(basic_port_allocator.cc:924): Port[6c03be30:0:1:0:local:Net[wlp3s0:192.168.0.x/24:Unknown:id=3]]: Gathered candidate: Cand[:3460680282:1:udp:2122129151:192.168.0.x:49773:local::0:l8iN:j+YbUepJBqspezc2DEz0GlZe:3:50:0]
(basic_port_allocator.cc:957): Port[6c03be30:0:1:0:local:Net[wlp3s0:192.168.0.x/24:Unknown:id=3]]: Port ready.
(basic_port_allocator.cc:1068): Port[6c03be30:0:1:0:local:Net[wlp3s0:192.168.0.x/24:Unknown:id=3]]: Port completed gathering candidates.
(audio_device_buffer.cc:288): Size of playout buffer: 960
(basic_port_allocator.cc:1360): Net[wlp3s0:2a02:908:1013:x:x:x:x:x/64:Unknown:id=4]: Allocation Phase=Relay
(basic_port_allocator.cc:1360): Net[wlp3s0:2a02:908:1013:x:x:x:x:x/128:Unknown:id=5]: Allocation Phase=Relay
(basic_port_allocator.cc:1360): Net[wlp3s0:192.168.0.x/24:Unknown:id=3]: Allocation Phase=Relay
(basic_port_allocator.cc:1360): Net[wlp3s0:2a02:908:1013:x:x:x:x:x/64:Unknown:id=4]: Allocation Phase=Tcp
(port.cc:186): Port[6c044f00::1:0:local:Net[wlp3s0:2a02:908:1013:x:x:x:x:x/64:Unknown:id=4]]: Port created with network cost 50
(basic_port_allocator.cc:885): Adding allocated port for 0
(basic_port_allocator.cc:906): Port[6c044f00:0:1:0:local:Net[wlp3s0:2a02:908:1013:x:x:x:x:x/64:Unknown:id=4]]: Added port to allocator
(basic_port_allocator.cc:924): Port[6c044f00:0:1:0:local:Net[wlp3s0:2a02:908:1013:x:x:x:x:x/64:Unknown:id=4]]: Gathered candidate: Cand[:370005403:1:tcp:1518283007:[2a02:908:1013:x:x:x:x:x]:54659:local::0:l8iN:j+YbUepJBqspezc2DEz0GlZe:4:50:0]
(basic_port_allocator.cc:957): Port[6c044f00:0:1:0:local:Net[wlp3s0:2a02:908:1013:x:x:x:x:x/64:Unknown:id=4]]: Port ready.
(basic_port_allocator.cc:1068): Port[6c044f00:0:1:0:local:Net[wlp3s0:2a02:908:1013:x:x:x:x:x/64:Unknown:id=4]]: Port completed gathering candidates.
(basic_port_allocator.cc:1360): Net[wlp3s0:2a02:908:1013:x:x:x:x:x/128:Unknown:id=5]: Allocation Phase=Tcp
(port.cc:186): Port[6c0460a0::1:0:local:Net[wlp3s0:2a02:908:1013:x:x:x:x:x/128:Unknown:id=5]]: Port created with network cost 50
(basic_port_allocator.cc:885): Adding allocated port for 0
(basic_port_allocator.cc:906): Port[6c0460a0:0:1:0:local:Net[wlp3s0:2a02:908:1013:x:x:x:x:x/128:Unknown:id=5]]: Added port to allocator
(basic_port_allocator.cc:924): Port[6c0460a0:0:1:0:local:Net[wlp3s0:2a02:908:1013:x:x:x:x:x/128:Unknown:id=5]]: Gathered candidate: Cand[:849103887:1:tcp:1518217471:[2a02:908:1013:x:x:x:x:x]:44099:local::0:l8iN:j+YbUepJBqspezc2DEz0GlZe:5:50:0]
(basic_port_allocator.cc:957): Port[6c0460a0:0:1:0:local:Net[wlp3s0:2a02:908:1013:x:x:x:x:x/128:Unknown:id=5]]: Port ready.
(basic_port_allocator.cc:1068): Port[6c0460a0:0:1:0:local:Net[wlp3s0:2a02:908:1013:x:x:x:x:x/128:Unknown:id=5]]: Port completed gathering candidates.
(basic_port_allocator.cc:1360): Net[wlp3s0:192.168.0.x/24:Unknown:id=3]: Allocation Phase=Tcp
(port.cc:186): Port[6c0473c0::1:0:local:Net[wlp3s0:192.168.0.x/24:Unknown:id=3]]: Port created with network cost 50
(basic_port_allocator.cc:885): Adding allocated port for 0
(basic_port_allocator.cc:906): Port[6c0473c0:0:1:0:local:Net[wlp3s0:192.168.0.x/24:Unknown:id=3]]: Added port to allocator
(basic_port_allocator.cc:924): Port[6c0473c0:0:1:0:local:Net[wlp3s0:192.168.0.x/24:Unknown:id=3]]: Gathered candidate: Cand[:2160473770:1:tcp:1518149375:192.168.0.x:43189:local::0:l8iN:j+YbUepJBqspezc2DEz0GlZe:3:50:0]
(basic_port_allocator.cc:957): Port[6c0473c0:0:1:0:local:Net[wlp3s0:192.168.0.x/24:Unknown:id=3]]: Port ready.
(basic_port_allocator.cc:1068): Port[6c0473c0:0:1:0:local:Net[wlp3s0:192.168.0.x/24:Unknown:id=3]]: Port completed gathering candidates.
(basic_port_allocator.cc:1143): All candidates gathered for 0:1:0
(p2p_transport_channel.cc:952): P2PTransportChannel: 0, component 1 gathering complete
```

**Comment by ilya-fedin** [comment]
> Telegram -debug output:

-debug argument doesn't control output, it creates logs in ~/.local/share/TelegramDesktop/DebugLogs

**Comment by granger75** [comment]
It seems it's fixed in 2.6 . Thanks al lot!

**Comment by banderlog** [comment]
Yep, in 2.6.1 problem is gone
